tensorflow/compiler/xla/stream_executor/rocm/rocm_driver.cc

```diff
@@ -1241,6 +1242,29 @@ static tsl::StatusOr<T> GetSimpleAttribute(hipDevice_t device,
   return true;
 }
 
+static uint64 GetReservedMemory() {
+  uint64 reserve = 0;
+  hipDeviceProp_t props;
+  hipDevice_t dev;
+  hipError_t res = wrap::hipGetDevice(&dev);
+  if(res == hipSuccess) {
```

Comment:
Would it be easier to read if we add early exits?

```
if (res != hipSuccess) {
  return 0;
}
```

Reply:
Thanks for the feedback! I have changed it as early exist.